Sources/BraintreeCore/Authorization/InvalidAuthorization.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+/// An invalid authorization type
+class InvalidAuthorization: ClientAuthorization {
+    
+    let type = AuthorizationType.invalidAuthorization
+    let configURL: URL
+    let bearer: String
+    let originalValue: String
```

Comment:
None of these values are actually going to be used / we're just setting these properties to conform to the protocol but never using them functionally. 

I crossed this bridge when messing around with this in [this branch](https://github.com/braintree/braintree_ios/compare/v7...spike-api-client-std-init). I wonder if there's a cleaner pattern.

Reply:
Potentially but I do think there is some value in enforcing them for the valid auth types. Since they won't be used it seems harmless. We also do something [similar on Android](https://github.com/braintree/braintree_android/blob/8d51b5da585260c740dfbdfeff322face580d81e/BraintreeCore/src/main/java/com/braintreepayments/api/core/InvalidAuthorization.kt#L4).